CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ position: 103
 Since v1.17.1, chproxy follows [semantic versioning](https://semver.org/).
 Don't expect breaking changes between 2 releases if they have the same major version.
 
+### <a id="229"></a> release 1.20.0, 2022-11-??
```

Comment:
We should also involve here the info about #115 as it'll be also involved within the 1.20.0

Reply:
cc @JingmaoYou

Reply:
until now the changelog only contains links to PR. It feels weird to link an issue that is not exactly what you fixed.